packages/react/src/components/OptionsMenu/OptionsMenuTrigger.tsx

```diff
@@ -19,6 +19,7 @@ function OptionsMenuTriggerComponent({
   return (
     <button
       type="button"
+      aria-haspopup="true"
```

Comment:
Let me do some testing here...I'm thinking we might want to `aria-haspopup="menu"` here